service/src/main/java/bio/terra/workspace/service/workspace/flight/WorkspaceFlightMapKeys.java

```diff
@@ -84,7 +84,7 @@ private ControlledResourceKeys() {}
 
     // AWS
     public static final String AWS_CLOUD_CONTEXT = "awsCloudContext";
-    public static final String AWS_ENVIRONMENT_USER_ROLE_ARN = "awsEnvironmentUserRoleArn";
```

Comment:
Not changing the key text here, since this is persisted in the database.

Reply:
How about keeping both? Not many notebooks are used now, and using the correct key would help later